registry.json

```diff
@@ -1,4 +1,19 @@
 [
+  {
+      "ticker":"GIBON",
+      "poolId":"b352315bd54d61dfcd8fd78a2aa4f3e0cc8ef342b9495465b0b0b2ce",
+      "social":{
+         "facebook":"",
+         "twitter":"@gibon_pool",
+         "website":"https://gibon.online",
+         "reddit":"",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"https://t.me/gibonpool",
```

Comment:
We need a user telegram handle, no groups of channels please. this is for voting purpose's.